### DISCUSSION TITLE
External links open in the same tab

### DISCUSSION THREAD

**Discussion by AnilSeervi** [mdn/yari#6049] [Polls]
Links tagged with the `external icon` open in the same tab. I personally think that this is bad UX.
The links should usually open as they represent, in another tab.

**Comment by OnkarRuikar** [comment]
I would like to open external links in new tabs on left click. But the middle mouse button does the job already. 
We can also use `CTRL` + `left click` and  `CTRL` + `ENTER` for the same.
By opening them in other tab on `left click`, we'll take away the choice of opening in same tab from the users. 

Mobile users are annoyed when links opened in new tabs, as they can’t use back button to return to the previous tab. And accessing the previous tab involves more taps.
